src/adt/edgeset.c

```diff
@@ -1224,6 +1224,42 @@ edge_set_find(const struct edge_set *set, unsigned char symbol,
 	return 0;		/* not found */
 }
 
+int
+edge_set_check_edges(const struct edge_set *set, unsigned char label,
+    fsm_state_t *to_state, uint64_t labels[256/64])
```

Comment:
I think introducing this means we're obliged to retire `USE_EDGE_BITSET`, because there's no equivalent implementation when that macro is not true

Reply:
done in https://github.com/katef/libfsm/pull/394